f4pga/flows/common.py

```diff
@@ -64,34 +64,14 @@ def with_qualifier(name: str, q: str) -> str:
         return decompose_depname(name)[0] + "!"
 
 
-_sfbuild_module_collection_name_to_path = {}
-
-
-def scan_modules(mypath: str):
-    global _sfbuild_module_collection_name_to_path
-    sfbuild_home = mypath
-    _sfbuild_module_collection_name_to_path = {
-        re_match("(.*)_modules$", moddir).groups()[0]: str(Path(sfbuild_home) / moddir)
-        for moddir in [dir for dir in os_listdir(sfbuild_home) if re_match(".*_modules$", dir)]
-    }
-
-
 def resolve_modstr(modstr: str):
     """
-    Resolves module location from modulestr.
+    Resolves module location from modstr.
```

Comment:
I guess this is a good opportunity to rephrase this comment.

```suggestion
    Resolves module location given its name.
```